src/tamp_improv/benchmarks/base.py

```diff
@@ -50,6 +50,7 @@ class BaseTAMPSystem(Generic[ObsType, ActType], ABC):
     def __init__(
         self,
         planning_components: PlanningComponents[ObsType],
+        n_blocks: int = 2,  # pylint: disable=unused-argument
```

Comment:
seems strange to include this in the base class -- why can't we just put it in the blocks system?